### ISSUE TITLE
Add more tests to common accessibility bridge unittest

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#77750]
The current accessibility bridge in common library does not have good test coverage. We have existing unittests
https://github.com/flutter/engine/blob/dcb57f003bd3f5f8fbb3c75cd94443e8096ade94/shell/platform/common/accessibility_bridge_unittests.cc

We should add more test to test different semantics update scenarios.
